src/lib/viewers/media/MediaBaseViewer.js

```diff
@@ -303,30 +303,28 @@ class MediaBaseViewer extends BaseViewer {
     }
 
     /**
-     * Determines if media should autoplay based on cached settings value.
+     * Handler for autoplay failure
+     * Overridden in child class
+     *
+     * @protected
+     */
+    handleAutoplayFail = () => {};
+
+    /**
+     * Autoplay the media
      *
      * @private
      * @emits volume
      * @return {Promise}
      */
     autoplay() {
         // Play may return a promise depending on browser support. This promise
-        // will resolve when playback starts. If it fails, we mute the video
-        // and try to play again.
+        // will resolve when playback starts.
         // https://webkit.org/blog/7734/auto-play-policy-changes-for-macos/
-        const autoPlayPromise = this.mediaEl.play();
+        const autoPlayPromise = this.play();
```

Comment:
This is legacy, but can we rename this to be `playPromise` or something like that? `autoPlayPromise` seems like a misnomer since the promise is from `play` itself